RecoBTag/SecondaryVertex/plugins/TemplatedSecondaryVertexProducer.cc

```diff
@@ -273,15 +278,16 @@ TemplatedSecondaryVertexProducer<IPTI,VTX>::TemplatedSecondaryVertexProducer(
 	    constraint == CONSTRAINT_BEAMSPOT ||
 	    constraint == CONSTRAINT_PV_PRIMARIES_IN_FIT )
 	    token_BeamSpot = consumes<reco::BeamSpot>(params.getParameter<edm::InputTag>("beamSpotTag"));
-        useExternalSV = false;
-        if(params.existsAs<bool>("useExternalSV")) useExternalSV = params.getParameter<bool> ("useExternalSV");
-        if(useExternalSV) {
+	useExternalSV = false;
+	if(params.existsAs<bool>("useExternalSV")) useExternalSV = params.getParameter<bool> ("useExternalSV");
```

Comment:
Isn't this syntax deprecated? Last time I was asked to explicitly ask for all the configuration parameters and create a cfi with all the parameters to default values